src/style.css

```diff
@@ -0,0 +1,92 @@
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0 8px;
```

Comment:
```suggestion
  margin-top: 0;
```

Reply:
default margin in body is 8px, you don't need to set it manually

Reply:
understood